src/App.tsx

```diff
@@ -1,61 +1,148 @@
+import classNames from 'classnames';
+
+import { useState } from 'react';
+import { TodoList } from './components/TodoList/TodoList';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
 export const App = () => {
+  const todosWithUsers = todosFromServer.map(todo => {
+    const chosen = usersFromServer.find(user => user.id === todo.userId);
+
+    if (!chosen) {
+      throw new Error(`error in ${todo.id} task - no one user was found for this task`);
+    }
+
+    return {
+      ...todo,
+      user: chosen,
+    };
+  });
+
+  const [chosenUserValue, setChosenUserValue] = useState('0');
+  const [newTitle, setNewTitle] = useState('');
+  const [listWithUsers, setListWithUsers] = useState(todosWithUsers);
+  const [addClicked, setAddClicked] = useState(false);
+
+  const createTask = () => {
+    if (!newTitle) {
```

Comment:
```suggestion
    if (!newTitle.trim().length) {
```

<img width="193" alt="image" src="https://user-images.githubusercontent.com/55802630/184825807-b263eec6-2881-4733-be92-2951e9cf70ee.png">

Reply:
fixed